doc/user/content/get-started/_index.md

```diff
@@ -12,16 +12,25 @@ menu:
     weight: 5
 ---
 
-Materialize is a **streaming database** purpose-built for low-latency
-applications. You can use it to process data at speeds and scales not possible
-in traditional databases, but without the cost, complexity, or development time
-of most streaming engines.
+Materialize is the Operational Data Warehouse that delivers the speed of
+streaming with the ease of a data warehouse. With Materialize, organizations can
+operate on real-time data just by using SQL.
```

Comment:
> With Materialize, organizations can operate on real-time data just by using SQL.

Not sure if we're planning to rehash our positioning as a follow-up, but wanted to mention that this isn't a differentiator in itself. We could maybe be consistent with the [blurb on our website](https://materialize.com/), rather than ride with a whole new definition?